client-spark/spark3/src/main/java/org/apache/spark/shuffle/writer/RssShuffleWriter.java

```diff
@@ -860,6 +860,11 @@ public Option<MapStatus> stop(boolean success) {
       // free all memory & metadata, or memory leak happen in executor
       if (bufferManager != null) {
         bufferManager.freeAllMemory();
+        try {
+          bufferManager.close();
```

Comment:
If bufferManager has the method `close`, the method `close` method should contain `freeAllMemory` method.

Reply:
sounds good